e2e/nextjs/app-router/app/package.json

```diff
@@ -10,7 +10,7 @@
     },
     "dependencies": {
         "@devcycle/nextjs-sdk": "file:../../../../dist/sdk/nextjs",
-        "next": "^15.2.3",
+        "next": "~15.2.4",
```

Comment:
do we care about going to 15.3 it was released 5 days ago

Reply:
`15.3` changed a bunch of core stuff in Next (again), so was avoiding it for now.